src/main/java/org/arquillian/container/chameleon/spi/model/Target.java

```diff
@@ -5,7 +5,7 @@
 public class Target {
 
     public static enum Type {
-        Remote, Managed, Embedded, Default;
+        Remote, Managed, Embedded, Domain, Default;
```

Comment:
Right. I was thinking about adding Domain_Managed, Domain_Remote, etc. but it didn't look very good. Specifying "domain" in the server part sounds better. Let me look at this.